docs/source/preparing_datasets/dataset_format.md

```diff
@@ -0,0 +1,25 @@
+# Dataset Format
+
+## Introduction
+
+To use StreamingDataset, one must convert raw data into one of our supported serialized dataset formats. With massive datasets, our serialization format choices are critical to the ultimate observed performance of the system. For deep learning models, we need extremely low latency cold random access of individual samples granularity to ensure that dataloading is not a bottleneck to training.
+
+StreamingDataset is compatible with any data type, including **images**, **text**, **video**, and **multimodal** data. StreamingDataset supports the following formats:
+ * MDS (Mosaic Data Shard, most performant), through {class}`streaming.MDSWriter`
```

Comment:
what does "most performant" mean?

it's possible to get marginally faster lookup with csv shards currrently

Reply:
most performant in terms of space and iteration time? is that not true?